keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -591,6 +670,14 @@ service Provisioner {
 This call is meant to retrieve the unique provisioner Identity. This identity will have to be set in `BucketRequest.Provisioner` field in order to invoke this specific provisioner.
 
 ```
+message Protocol {
+	oneof type {
+		S3Parameters s3 = 1;
+		AzureBlobParameters azureBlob = 2;
```

Comment:
Nit: there's a bit of a discrepancy here, where `Blob` is called `AzureBlob`, and `S3` is not called, well, `AWSS3`. Not sure what the GCP non-branded equivalent would be.